test/application_system_test_case.rb

```diff
@@ -3,6 +3,7 @@
 require 'test_helper'
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  include ApplicationHelper
```

Comment:
Suggest including this in the specific tests class where helpers are needed rather than `ApplicationSystemTestCase`.